src/inventory-service/inventory-core/src/main/java/com/inventory/core/config/AppConfig.java

```diff
@@ -0,0 +1,103 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2024 Datadog, Inc.
+ */
+
+package com.inventory.core.config;
+
+import jakarta.enterprise.context.ApplicationScoped;
+
+import java.util.Optional;
+
+import org.eclipse.microprofile.config.inject.ConfigProperty;
+
+/**
+ * Centralized application configuration using Quarkus config properties
```

Comment:
Quarkus config properties are indeed the thing to use (and you can do env overrides anyhow). 

Any reason you need to centralize this?

Reply:
Yeah, exactly. Quick glance at stuff, and some of the core stuff is actually shared between repos. I could split some of it down to be fair though.